sprite.go

```diff
@@ -746,12 +745,12 @@ func (p *Sprite) Step__2(step float64, animname string) {
 	if ani, ok := p.animations[animname]; ok {
 		anicopy := *ani
 		anicopy.From = 0
-		anicopy.To = step * float64(p.g.stepUnit)
+		anicopy.To = step * float64(p.g.gridUnit)
```

Comment:
anicopy.To should be `step`. here we use `world axis`.

Reply:
this time compatible first,  the next pr change